multiclusterhub/charts/insights-chart/templates/insights-client-service.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: insights-client-certs
+  name: insights-client
+  labels:
+    app: {{ template "insights.name" . }}
+    chart: {{ template "insights.chart" . }}
+    component: insights-client
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  ports:
+    - port: 3030
+      targetPort: 3030
+      protocol: TCP
+      name: http
+  selector:
+    app: {{ template "insights.name" . }}
+    component: "insights-client"
+    release: {{ .Release.Name }}
+  sessionAffinity: ClientIP
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822